ui/src/components/VolumeMetricGraphCard.js

```diff
@@ -161,20 +156,26 @@ const MetricGraphCard = (props) => {
 
   // slot[0] => timestamp
   // slot[1] => value
-  const volumeUsageData = volumeUsedOperated?.map((slot) => {
+  const volumeUsageData = volumeUsageOperated?.map((slot) => {
     return {
       date: new Date(slot[0] * 1000), // convert from the RFC 3339 to time in JS
```

Comment:
Instead of having a comment (which is not repeated l.176 and l.168), we could have a little function `dateFromRFC3339` that you could reuse everywhere it's needed.
And thanks to the explicit name, you won't need the comment.